src/main/java/com/cburch/logisim/gui/log/LogFrame.java

```diff
@@ -50,15 +50,12 @@
 import java.awt.Dimension;
 import java.awt.Rectangle;
 import java.awt.Toolkit;
+import java.awt.event.KeyEvent;
 import java.awt.event.WindowAdapter;
 import java.awt.event.WindowEvent;
 import java.util.HashMap;
 import java.util.Map;
-import javax.swing.JButton;
-import javax.swing.JFrame;
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
-import javax.swing.JTabbedPane;
+import javax.swing.*;
```

Comment:
Personally, I prefer to explicitly import only the used classes.

Reply:
Fixed

Reply:
Thanks!